crates/sui-protocol-config/src/lib.rs

```diff
@@ -3076,6 +3083,13 @@ impl ProtocolConfig {
 
                     cfg.validator_validate_metadata_cost_base = Some(20000);
                 }
+                71 => {
+                    if chain != Chain::Mainnet {
```

Comment:
This is just being conservative.  If rolling out to mainnet at the same time is acceptable then let's do it!

Reply:
Ok. After looking at the related logic a bit, it seems there are some interactions between gas price and max_tx_gas limit, among other effects of using high gas price. I think being a bit conservative is warranted here. We should definitely try out using very high gas price on testnet first before changing the limit on mainnet. We may need to update max_tx_gas or switch to a different parameter limiting gas unit instead of total gas. They can be done separately.

Reply:
Yes I have justed realized there is `max_tx_gas` as well.
Tried to change it locally and that has just caused so many tests to fail (these tests use `max_tx_gas` as gas budget if not explicitly set).  Looks like we are going to need another PR for it.

Reply:
Agree let's not change `max_tx_gas` in this PR. In fact we can postpone changing `max_gas_price` too if you think just changing it will not be useful. I have asked experts on the gas logic for the best way forward. cc @dariorussi @tzakian

Reply:
please take a look at my main comment